modules/docs/arrow-docs/docs/docs/arrow-core/typeclasses/monadlogic/README.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: docs-core
+title: MonadLogic
+permalink: /docs/arrow/typeclasses/monadlogic/
+redirect_from:
+  - /docs/typeclasses/monadlogic/
+---
+
+## MonadLogic
+
+The `MonadLogic` typeclass provides some functions to control when computations should be performed.
```

Comment:
A much better explanation would be to show why and where [MonadPlus] is unfair (left-biased usually) and how [MonadLogic] improves that. In general mentioning fairness is always good here.